Mods/interface/PuppiMod.h

```diff
@@ -23,10 +23,6 @@ namespace mithep
                 const char *title="Puppi module" );
      ~PuppiMod();
 
-      Int_t GetParticleType( const PFCandidate *cand );
```

Comment:
These functions are used by the module internally, so better be hidden (i.e. made private). Otherwise your job macro can accidentally call e.g. GetEtaBin, which relies on eta binning set up already, and can segfault. Of course that's a stupid example, but just as a matter of principle and ease of code reading, make a clear distinction between public and private members.